extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/PersistenceUnitDescriptorBuildItem.java

```diff
@@ -79,6 +79,10 @@ public boolean isFromPersistenceXml() {
         return fromPersistenceXml;
     }
 
+    public boolean isReactive() {
+        return isReactive;
+    }
```

Comment:
Maybe you could remove this field from here, since you added the information to the `QuarkusPersistenceUnitDescriptor`, and you have access to the descriptor here.

Reply:
Yes, removed